src/_locales/en/messages.json

```diff
@@ -48,7 +48,7 @@
     "message": "New"
   },
   "passff_button_root_label": {
-    "message": "Display root"
+    "message": "Display all"
```

Comment:
I don't agree with this: Displaying "all" password entries is different from displaying the "root" (directory) of the password store.

Reply:
Oh yes I wasn't aware of this.
Displaying all passwords is rather a recursive list (think `ls -R`).
This button just displays the top-level directory.
Maybe `top-level`?
Example: Top-level Domain.